docs/2.guide/3.going-further/9.debugging.md

```diff
@@ -38,6 +38,10 @@ It is possible to debug your Nuxt app in your IDE while you are developing it.
 
 You may need to update the config below with a path to your web browser. For more information, visit the [VS Code documentation about debug configuration](https://go.microsoft.com/fwlink/?linkid=830387).
 
+::important
+Make sure you have shamefully-hoist enabled if you use pnpm to properly start your server in debug mode with below configuration.
```

Comment:
Wouldn't it be better to suggest adding `nuxi` to devDependencies?

Reply:
AH yes thats a cleaner solution. I didn't come up with that before. 
Would: 
> _Make sure you have nuxi installed as Devdependency if you use pnpm to properly start your server in debug mode with below configuration._ 

suffice?

Reply:
That sounds good!

> If you use `pnpm`, you will need to have `nuxi` installed as a devDependency for the configuration below to work.